src/agents/dj-personas.ts

```
@@ -0,0 +1,227 @@
+        case 'quick_cut':
+          transitionType = bpmDifference > 10 ? 'cut' : 'fade';
+          mixPoint = fromTrack.duration * 0.9; // Later mix for quick transition
+          break;
+        case 'scratch':
+          transitionType = 'cut'; // Always cut for scratch transitions
+          mixPoint = fromTrack.duration * 0.95;
+          break;
+        case 'creative':
+          // Random creative transitions
+          const types: MixDecision['transitionType'][] = ['beatmatch', 'cut', 'fade', 'effect'];
+          transitionType = types[Math.floor(Math.random() * types.length)];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)